portfolio/src/main/webapp/index.html

```diff
@@ -102,9 +102,16 @@ <h1><span class="text-emphasis">Antonio</span>'s Portfolio</h1>
         </div>
       </div>
       <div id="comments">
+        <form action="/data" method="POST">
+          <p>Enter a comment:</p>
```

Comment:
This can probably be a `<label>` for the textarea

Reply:
Didn't know about labels before, thank you!